codegenerator/cli/templates/static/codegen/src/Benchmark.res

```diff
@@ -290,18 +334,22 @@ module Summary = {
         summaryData
         ->Js.Dict.get(eventProcessingGroup)
         ->Option.flatMap(g => g->Js.Dict.get(batchSizeLabel))
-        ->Option.map(data => data->Array.reduce(0., (acc, d) => acc +. d))
-        ->Option.getWithDefault(0.)
+        ->Option.map(data => data.sum)
+        ->Option.getWithDefault(BigDecimal.zero)
 
       let totalRuntimeMillis =
         millisAccum.endTime->Js.Date.getTime -. millisAccum.startTime->Js.Date.getTime
 
       let totalRuntimeSeconds = totalRuntimeMillis /. 1000.
 
-      let eventsPerSecond = (batchSizesSum /. totalRuntimeSeconds)->round(~precision=2)
+      let eventsPerSecond =
+        batchSizesSum
+        ->BigDecimal.div(BigDecimal.fromFloat(totalRuntimeSeconds))
+        ->BigDecimal.decimalPlaces(2)
```

Comment:
Should we have decimals here?

Reply:
It doesn't really matter I suppose. I just kept it at 2.